tests/Knp/Menu/Tests/Twig/MenuExtensionTest.php

```diff
@@ -20,173 +20,201 @@ final class MenuExtensionTest extends TestCase
     public function testRenderMenu(): void
     {
         $menu = $this->getMockBuilder(ItemInterface::class)->getMock();
-        $helper = $this->getHelperMock(['render']);
-        $helper->expects($this->once())
+        $helperMock = $this->getHelperMock(['render']);
+        $helperMock->expects($this->once())
             ->method('render')
             ->with($menu, [], null)
             ->willReturn('<p>foobar</p>')
         ;
+        /** @var Helper&MockObject $helper */
+        $helper = $helperMock;
+        $matcher = null;
+        $manipulator = null;
 
-        $this->assertEquals('<p>foobar</p>', $this->getTemplate('{{ knp_menu_render(menu) }}', $helper)->render(['menu' => $menu]));
+        $this->assertEquals('<p>foobar</p>', $this->getTemplate('{{ knp_menu_render(menu) }}', $helper, $matcher, $manipulator)->render(['menu' => $menu]));
     }
 
     public function testRenderMenuWithOptions(): void
     {
         $menu = $this->getMockBuilder(ItemInterface::class)->getMock();
-        $helper = $this->getHelperMock(['render']);
-        $helper->expects($this->once())
+        $helperMock = $this->getHelperMock(['render']);
+        $helperMock->expects($this->once())
             ->method('render')
             ->with($menu, ['firstClass' => 'test'], null)
             ->willReturn('<p>foobar</p>')
         ;
+        /** @var Helper&MockObject $helper */
+        $helper = $helperMock;
```

Comment:
Why renaming the variable instead of working with the same name from the start ?